.github/workflows/chart-testing.yaml

```diff
@@ -37,6 +37,8 @@ jobs:
 
       - name: Set up chart-testing
         uses: helm/chart-testing-action@v2.0.1
+        with:
+          version: 3.5.2
```

Comment:
Latest chart-testing version is only 3.3.1 (https://github.com/helm/chart-testing/releases/tag/v3.3.1). We probably don't need to override this here though, and just upgrade as the chart-testing-action upgrades.
```suggestion
```

Reverting this will at least fix the current error in the checks.